Commonalities/documentation/Glossary.md

```diff
@@ -21,6 +22,9 @@
 | **Longitude** | | longitude |
 | **accuracy** | | accuracy |
 | **Location Verification result** | As value is a boolean, it matches better with an adjective. In future versions we may need to enhance current response model to give mor information about the verification process, for example to deal with partial matches. |locationVerified |
+| **Telco Operator Edge site** | Edge Datacenter where the Application are deployed. Term for the documentation. |N/A|N/A |
+| **API Producer** | Software component, inside the platform, that exposes the CAMARA API. Term for the documentation. |N/A|N/A |
```

Comment:
Only Postman uses the term "API Producer". The more usual term is "API Provider". This term should at least be allowed as an alternative, if not adopted as the preferred terminology.

Reply:
It came to me out of habit because in 3GPP SA5 we define Management Service **Producers**. Just to be clear, we are not talking about the entity that exposes the API, e.g. Google, but about the software component generating/producing /providing :-) the API. I did some further research and both seems to me appropriate. 

We can put both. we just need to decide which is the main one  (the other is the alternative). I'm fine with any decision.

Reply:
@eric-murray I have introduced API Provider as alternative friendly name. Is this fine? can we merge?